src/RiotAPILibraries.ts

```diff
@@ -181,7 +181,7 @@ export default class RiotAPILibraries {
 
         // Check if alias
         for (const [key, values] of Object.entries(this.settings.riotApiLibraries.aliases)) {
-            if (values.find(self => self.toLowerCase() === language.toLowerCase())) {
+            if (values.find(self => self.toLowerCase() === language)) {
```

Comment:
Whats the reason behind removing the .toLowerCase() here?

Reply:
https://github.com/Querijn/BottyMcBotface/blob/0f6cd50bee4775dfefe180a064125f1d1ef70cb2/src/RiotAPILibraries.ts#L85

It's already made into lowercase when it's selected.